src/components/organisms/Dashboard/hooks.ts

```diff
@@ -117,25 +114,23 @@ export default (teamId?: string) => {
     notifyOnNetworkStatusChange: true,
   });
 
-  const projectNodes = projectData?.projects.edges.map(e => e.node) as ProjectNodes;
-
   const projects = useMemo(() => {
-    return (projectNodes ?? [])
+    return (projectData?.projects.edges ?? [])
       .map<Project | undefined>(project =>
-        project
+        project.node
           ? {
-              id: project.id,
-              description: project.description,
-              name: project.name,
-              image: project.imageUrl,
-              status: toPublishmentStatus(project.publishmentStatus),
-              isArchived: project.isArchived,
-              sceneId: project.scene?.id,
+              id: project.node.id,
+              description: project.node.description,
+              name: project.node.name,
+              image: project.node.imageUrl,
+              status: toPublishmentStatus(project.node.publishmentStatus),
+              isArchived: project.node.isArchived,
+              sceneId: project.node.scene?.id,
             }
           : undefined,
       )
       .filter((project): project is Project => !!project);
-  }, [projectNodes]);
+  }, [projectData?.projects.edges]);
```

Comment:
Similar to the comment above, you can destructure the node so you can clean up the assignment below, something like this:

```
 const projects = useMemo(() => {
    return (projectData?.projects.edges ?? [])
      .map<Project | undefined>(({node})=>
           node
          ? {
              id: node.id,
              description: node.description,
              name: node.name,
              image: node.imageUrl,
              status: toPublishmentStatus(node.publishmentStatus),
              isArchived: node.isArchived,
              sceneId: node.scene?.id,
            }
          : undefined,
      )
      .filter((project): project is Project => !!project);
  }, [projectData?.projects.edges]);
```
You can also destructure further like this:
```
(projectData?.projects.edges ?? [])
      .map<Project | undefined>(({node: { id, description, name, imageUrl: image, publishmentStatus: status, isArchived, scene: {id?: sceneId}}})=> {
   id,
   description,
   name,
   etc, 
   etc,
}
```
The second one might be too much? Also, you can go even further to ONLLY destructure specifically the keys that are different between the data and the frontend's Project (aka image VS imageUrl, etc) and leave the rest as `...rest`. But again, it might be too much and the code might become harder to understand. I'll let you play around and find whats best here